lms/djangoapps/oauth2_handler/handlers.py

```diff
@@ -53,6 +53,7 @@ class ProfileHandler(object):
 
     def scope_profile(self, _data):
         """ Add specialized claims. """
+        # TODO: Determine if Ecommerce ever uses this.
```

Comment:
Reviewer: can this ever be called in ecommerce?

Reply:
Not anymore during the SSO login path.
Not sure about other API paths.  But my guess is not.

Reply:
Thanks.  I dropped the comment.